src/DynamoCoreWpf/Controls/StandardPanel.xaml.cs

```diff
@@ -47,7 +47,18 @@ private void OnHeaderButtonClick(object sender, RoutedEventArgs e)
             e.Handled = true;
         }
 
-        private void OnMouseLeftButtonDown(object sender, MouseButtonEventArgs e)
+        private void OnMouseMove(object sender, MouseEventArgs e)
+        {
+            if (e.LeftButton != MouseButtonState.Pressed)
+                return;
+
+            var listBoxItem = sender as ListBoxItem;
+            var searchElementVM = listBoxItem.DataContext as NodeSearchElementViewModel;
+
+            DragDrop.DoDragDrop(listBoxItem, new DragDropNodeSearchElementInfo(searchElementVM.Model), DragDropEffects.Copy);
+        }
+
+        private void OnMouseLeftButtonUp(object sender, MouseButtonEventArgs e)
```

Comment:
Great! Having `up` create a node is better than `down` (just to align with button presses throughout the system).